scripts/e2e_check_pod_restarts.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+restarts=$(kubectl get pods -n mayastor | grep -e mayastor -e moac | awk '{print $4}')
+for num in $restarts
+do
+    if [ "$num" -ne "0" ]
+    then
```

Comment:
Looks more consistent with 'if' and 'then' on the same line, as in the other script above.

Reply:
Done